src/ansys/mapdl/core/launcher.py

```diff
@@ -1507,10 +1507,10 @@ def launch_mapdl(
             mapdl = MapdlConsole(loglevel=loglevel, log_apdl=log_apdl, **start_parm)
         elif mode == "corba":
             try:
-                # pending deprication to ansys-mapdl-corba
+                # pending deprecation to ansys-mapdl-corba
                 from ansys.mapdl.core.mapdl_corba import MapdlCorba
-            except ImportError:
-                raise ImportError(
+            except ModuleNotFoundError:
```

Comment:
There's no way we're going to cover corba for the moment.

```suggestion
            except ModuleNotFoundError:  # pragma: no cover
```